File: PP31.cpp
 

#include<iostream>
using namespace std;

class Pattern
{
    private:
        int iRow;

    public:
        Pattern(int a)
        {
            this->iRow = a;
           
        }

         void DisplayPattern()
        {
			int iCnt = 0;
			char ch = '\0';
                for(iCnt = 1,ch = 'A'; iCnt <= iRow;iCnt++, ch++)
				{
					cout<<ch<<endl;
				}
        }
};

int main()
{
    Pattern obj(5);
    obj.DisplayPattern();

    return 0;
}

File: PP31.java
// Template for probplems on Pattern printing

import java.lang.*;
import java.util.*;

class Pattern
{
    private int iRow, iCol;

    public Pattern(int a, int b)
    {
        iRow = a;
        iCol = b;
    }

    public void DisplayPattern()
    {
        int i = 0;
		char ch = '\0';
		
		for(i =1,ch ='A';i<=iRow;i++,ch++ )
		{
			System.out.println(ch);
		}
    }
}

class PP31
{
    public static void main(String arg[])
    {
            Pattern obj = new Pattern(5,5);

            obj.DisplayPattern();
    }
}